frontend/components/timeSeries/TimeSeries.js

```diff
@@ -1,18 +1,25 @@
 import React, {useState} from "react";
-import UploadFileInput from "../inputs/UploadFileInput";
+import UploadTimeSeriesFileInput from "../inputs/UploadTimeSeriesFileInput";
 
 const TimeSeries = () => {
     const [instrumentSamples, setInstrumentSamples] = useState(null);
 
     return (
         <>
             <h2>Time Series Data</h2>
-            <UploadFileInput
+            <UploadTimeSeriesFileInput
                 id={0}
                 uploadSuccessfulCallback={setInstrumentSamples}
                 apiEndpoint={'api/generate_instrument_2d/'}
             />
-            <p>Upload a .csv file where each row represents the ratios for a single sound.</p>
+            <p>Upload a .csv file with <code>m</code> rows and <code>n</code> columns, and
```

Comment:
Right now there are only three columns that you allow to be tuned (in the `UploadTimeSeriesFileInput` component above). Maybe that should be the max, communicated here?